elasticsearch.yml.tpl

```diff
@@ -1,29 +1,15 @@
 cluster:
   name: ${cluster_name}
 
-node:
```

Comment:
Tuhle část bych nemazal, potom nemáme jak nastavit role toho nodu,, chtělo by spíš přidat parametry pro nastavení.

Reply:
Tohle bych řešila už v rámci nějakého jiného PR, nebo až to bude potřeba... Teď to nastavitelné nebylo a ta konfigurace tam byla k ničemu.

Reply:
A já bych zrovna nemazal věc, která je v tom configu důležitá!

Reply:
Ta věc je v tom configu naprosto zbytečná, jen to kopíruje defaultní config.